micropython/build.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+set -e
+
+
+UPYTH="micropython-master"
+
+PREFIX_UPYTH="${TOPDIR}/phoenix-rtos-ports/micropython"
+PREFIX_UPYTH_BUILD="${PREFIX_BUILD}/micropython"
+PREFIX_UPYTH_PATCHSET="${PREFIX_UPYTH}/${UPYTH}-patchset"
+
+
+b_log "Building micropython"
+pushd .
+cd $PREFIX_UPYTH
+
+#
+# Download
+#
+if [[ ! -d ${UPYTH} ]]
+then
+	rm -rf "${UPYTH}" && rm -rf "${PREFIX_UPYTH_PATCHSET}"
+	git clone https://github.com/micropython/micropython.git ${UPYTH}
+	(cd ${UPYTH} && git reset --hard)
+
+	#Submodules bloat the size of micropython source code (up to several GB!). 
+	#Until submodules config feature is absent this line should not be uncommented!
+	#(cd ${UPYTH} && && git submodule update --init --recursive)
```

Comment:
There are two ways of downloading repository:
-using git as in code
-using wget

MicroPython heavily relies on submodules so the good side of using git would be ability to download with submodules if needed (and after addition of download config).
However for now wget may be faster and easier to just download code and unpack it client-side